build/ci-tf-e2e.Dockerfile

```diff
@@ -15,7 +15,7 @@ ENV PATH="/usr/local/go/bin:${PATH}"
 ENV GOPATH=/usr/local/go
 ENV TEST_OFFLINE_TOKEN=""
 
-# terraform-provider-rhcs repo
+# terraform-cprovider-rhcs repo
```

Comment:
Why this change? (this affects our CI as well)

Reply:
By a mistake - thanks I fixed it